.github/dependabot.yml

```diff
@@ -12,3 +12,9 @@ updates:
       interval: "weekly"
     commit-message:
       prefix: "chore"
+  - package-ecosystem: "docker"
+    directory: "/manifest_staging"
```

Comment:
The check unfortunately isn't recursive, so you'll need to manually specify the actual directory containing the chart... I filed https://github.com/dependabot/dependabot-core/issues/6047 for us to evaluate changing that behavior.

Reply:
And looks like there's another bug, even when you _do_ get it configured correctly: https://github.com/dependabot/dependabot-core/issues/6048

I'll try to debug ☝️ tomorrow.

Reply:
@jeffwidman Do you still want me to update the path and point it to the `values.yaml` directory?